pilot/pkg/features/pilot.go

```diff
@@ -328,4 +328,16 @@ var (
 			"No need to configure this for root certificates issued via Istiod or web-PKI based root certificates. "+
 			"Use || between <trustdomain, endpoint> tuples. Use | as delimiter between trust domain and endpoint in "+
 			"each tuple. For example: foo|https://url/for/foo||bar|https://url/for/bar").Get()
+
+	EnableDefaultToSystemCAValidation = env.RegisterBoolVar("USE_SYSTEM_CA_FALLBACK", false,
```

Comment:
Are you planning to keep two env vars, one for system CA and another for SAN verification?
Should we consolidate it as one flag INSECURE_SKIP_VERIFY?

Reply:
The idea behind having two env var is making life easier for the end user. If we use only one env var, user has to compute the location of their system CA files(which is probably the same for 90% of the cases). I don't have any strong feelings using either